Flexible Learning/units_manage_edit.php

```diff
@@ -98,9 +98,14 @@
         $row->addLabel('blurb', __('Blurb'));
         $row->addTextArea('blurb')->required();
 
+    $highestManageAction = getHighestGroupedAction($guid, '/modules/Flexible Learning/units_manage.php', $connection2);
+
     $row = $form->addRow();
         $row->addLabel('gibbonPersonIDCreator', __('Author'));
-        $row->addSelectUsers('gibbonPersonIDCreator')->photo(true, 'small');
+        $row->addSelectUsers('gibbonPersonIDCreator')
```

Comment:
There does not seem to be a change to units_manage_editProcess.php, but I guess there should be, as there is with addProcess